server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.use(bodyParser.json());
+
+app.post('/api/login', (req, res) => {
+  const { email, password } = req.body;
+  
+  // Add authentication logic here
+  
+  res.status(200).end();
+});
+
```

Comment:
Consider handling errors and adding appropriate HTTP status codes in the responses. This would improve the robustness of your server.